content/en/blog/_posts/2024-XX-XX-websocket-transition.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
+date: 2024-XX-XX
+slug: websockets-transition
+author: >
+  [Sean Sullivan](https://github.com/seans3) (Google)
+---
+
+With Kubernetes 1.31, PortForward over Websockets is moving to beta.
+RemoteCommand over Websockets previously graduated to beta in v1.30.
+
+Today’s article is about streaming API’s and the protocols used to implement
+these API’s.
+
+## Why have we modified the streaming protocol ?
+
+Consider the following situation: you’ve configured a proxy or gateway in front
+of your Kubernetes cluster, but you’ve discovered four kubectl commands no
+longer work: kubectl exec, kubectl cp, kubectl attach, and kubectl port-forward.
+For these kubectl commands, the HTTP REST requests are upgraded to streaming
+connections. But the streaming protocol used for these upgraded connections is
+SPDY/3.1, which has been **deprecated for eight years**. Your newly configured
+proxy or gateway does not support this out-of-date streaming protocol. As of the
+v1.31 release, however, by default all four of these kubectl commands now
+implement the more modern [Websocket streaming protocol](https://datatracker.ietf.org/doc/html/rfc6455).
```

Comment:
I would focus more on standardization than in age, as SPDY was never standardized and websockets is widely used standard